RealEstate/apps/core/templates/core/realtorHome.html

```diff
@@ -1,5 +1,56 @@
 {% extends "core/base.html" %}
 
-{% block body %}
-  <p>Home page for realtors.</p>
+{% block style %}
+<style>
+.vcenter {
+	display: inline-block;
+	vertical-align: middle;
+	float: none;
+	margin-left:-3px;
+}
+</style>
 {% endblock %}
+
+{% block body %}
+    <h2><strong> Dashboard </strong> </h2>
+    <ul class="list-group">
+        {% for couple, homebuyers in couples %}
+        <li class="list-group-item">
+			<div class="row">
+				<div class="col-sm-6 vcenter">
+				    {% for homebuyer in homebuyers %}
+				        <h4>{{homebuyer}}</h4>
+	                {% endfor %}
+				</div>
+				<div class="col-sm-4 vcenter">
+				    Right Holding Block
+				</div>
+				<div class="col-sm-2 vcenter">
+					<a href="homebuyerreport">Report</a>
```

Comment:
Look at other examples of how to link to urls contained in urls.py. This isn't quite correct.